syncserver/__init__.py

```diff
@@ -123,6 +126,42 @@ def itworks(request):
     config.add_view(itworks, route_name='itworks')
 
 
+def import_settings_from_environment_variables(settings, environ=None):
```

Comment:
Allow the most important settings to come from the environment, so we can run without a .ini config file if necessary.